flink/v1.14/flink/src/main/java/com/netease/arctic/flink/write/FlinkTaskWriterBuilder.java

```diff
@@ -60,7 +60,7 @@ public class FlinkTaskWriterBuilder implements TaskWriterBuilder<RowData> {
   private int partitionId = 0;
   private long taskId = 0;
   private RowType flinkSchema;
-  private int mask;
+  private long mask;
```

Comment:
Why change to long primitive type?

Reply:
There is originally long, and this method is invoked by the parameter long type everywhere.